src/tests/ftest/dfuse/fio_small.yaml

```diff
@@ -86,3 +86,5 @@ fio:
     numjobs: 1
 dfuse:
   mount_dir: "/tmp/daos_dfuse"
+  thread_count: 8
+  bind_cores: '0-3'
```

Comment:
Should this be dfuse_cores?   
from 
self.dfuse_cores = self.params.get('dfuse_cores', self.dfuse.namespace, None)

Reply:
Yes.